docs/documentation/plugins/index.html

```diff
@@ -1482,6 +1482,44 @@ <h3 id="skeleton">Skeleton</h3>
         }
     }
 
+    // If the plugin is a button
+    function buildButtonIcon() {
+        if (!document.querySelector("#trumbowyg-myplugin")) {
+            /*
+            When your button is created, an SVG will be inserted with an xref
+            to a symbol living at the fragment "#trumbowyg-myplugin".
+            For Trumbowyg's own plugins, this will come from a sprite sheet
+            which is injected into the document, built from the icon SVG
+            in "ui/icons" in your plugin's source tree.
+
+            But, nothing says it *has* to come from Trumbowyg's injected
+            sprite sheet; it only requires that this symbol exists in your
+            document. To allow stand-alone distribution of your plugin, we're
+            going to insert a custom SVG symbol into the document with the
+            correct ID.
+            */
+            const iconWrap = $(document.createElementNS("http://www.w3.org/2000/svg", "svg"));
+
+            /*
+            This class is only here to allow CSS to hide it. In your plugin's
+            stylesheet you will probably want something like this:
+
+            .trumbowyg-myplugin-icons {
+                display: none;
+            }
+            */
```

Comment:
Maybe I should add a generic class like that to the trumbowyg CSS file, allowing it to be shared by the default SVG *and* all additional ones :)

Reply:
I think that is a good idea. Is it a blocker for this PR?